src/System.Text.Json.EnumExtensions/Serialization/JsonStringEnumConverterWithAttributeSupport.cs

```diff
@@ -44,6 +44,19 @@ public class JsonStringEnumConverterWithAttributeSupport : JsonConverterFactory
                 _converterOptions |= EnumConverterOptions.ParseDescriptionAttribute;
             }
         }
+        
+        /// <summary>
+        /// Parameterless constructor.
+        /// </summary>
+        /// <param name="namingPolicy">Naming policy for writing enum values [Optional.</param>
```

Comment:
This comment about these 5 parameters can be removed?
Because these cannot be set when using this default empty constructor?

Correct ?

Reply:
@hughesjs 
Can you take a look at this comment?

Reply:
Yeah sorry, I'm away from home at the moment, will sort it when I'm back

Reply:
Just done it in the browser, that should do it